gui/qt/main_window.py

```diff
@@ -1839,6 +1873,18 @@ def spend_coins(self, coins):
         self.show_send_tab()
         self.update_fee()
 
+    def spend_crowdfunding(self, coins):
+        self.set_pay_from(coins)
+        self.show_crowdsale_input_button.setVisible(True)
+        self.send_button.setVisible(False)
+        self.preview_button.setVisible(False)
+        self.fee_e.setVisible(False)
+        self.show_send_tab() 
+        self.max_button.setVisible(False)
+        self.fee_slider.setVisible(False) 
+        self.fee_e_label.setVisible(False)
+        self.crowdfunding_in_process=True
+
```

Comment:
The UX is wonky.  If you specify a smaller amount than the actual coin amount (which the send tab lets you do), then the difference is interpreted as "fee", and it throws an error about ExcessiveFee in some cases.

Not sure what to do here, yet, about that -- but just noting it in this review.

Reply:
Is this really an issue?  You should never be doing this in practice.  The fund raise will always be larger than any individual UTXO.

Reply:
How would the amount be pre-filled if the wallet doesn't have knowledge about the fundraise?

Reply:
Oh so you sign on destination amount not utxo amount. Gotcha.

Well at any rate the UX sucks. Need a custom screen to make this clear.

I’m a programmer and it’s confusing as hell what’s going on.

The UI needs to be crystal clear.

Right now if you enter a wrong amount you get bizarro errors about excessive fee.

It’s just crappy.

We can do better.

Reply:
Also going forward we should think about a custom url  type: 

`bchcrowdfund:`

And have forms pre-filled in with values relevant to the crowdfund.

Reply:
1000% for the send tab being removed and replaced with a customisable base dialog that supports both direct instantiation by other things (like scheduled payments tab) and also inheritance for extended uses that keeps the base  version simple.